### PR TITLE
yamllint formatting cleanup for role ocp4-pam-fraudmanagement-workshop

### PR DESCRIPTION
**Pull request by ritesh97-rh**
##### SUMMARY
For the specified roles:
- Create a yamllint file to enforce formatting rules
- Make changes to existing code base to adhere to formatting requirements
- yamllint errors will be displayed during formatting checks in future development

##### ISSUE TYPE
- Code Cleanup Request

##### COMPONENT NAME
ocp4-workloadpam-fraudmanagement-workshop

### CODE REVIEW

File: ansible/roles/ocp4-workload-pam-fraudmanagement-workshop/tasks/workload_per_project_amqstreamsworkload.yml [hunk 1]
```diff
@@ -13,25 +13,51 @@
 
 - name: Get Kafka template
   uri:
-    url: "https://{{ gitea_route.stdout }}/{{ _namespace }}/proactive-fraud-detection-kafka/raw/branch/master/.openshiftio/application.yaml"
+    url: >-
```
Comment (stencell) [inline ansible/roles/ocp4-workload-pam-fraudmanagement-workshop/tasks/workload_per_project_amqstreamsworkload.yml:16]: For these, either increase the max to keep them on one line or use something like this. The way it is currently written is a bit hacky and hard to read.

```
url: >-
  {{ 'https://{0}/{1}/proactive-fraud-detection-kafka/raw/branch/master/.openshiftio/application.yaml'.format(
  gitea_route.stdout, _namespace) }}
```

Same with anything else in this task list that has things broken up like this.

File: ansible/roles/ocp4-workload-pam-fraudmanagement-workshop/tasks/workload_per_project_businessautomationworkload.yml [hunk 2]
```diff
@@ -166,10 +166,15 @@
 
 - name: Maven install for project {{ _case_kie_project_id }}
   uri:
-    url: "https://{{ businesscentral_host.stdout }}/rest/spaces/workshop/projects/{{ _case_kie_project_id }}/maven/install"
+    url: >-
```
Comment (stencell) [inline ansible/roles/ocp4-workload-pam-fraudmanagement-workshop/tasks/workload_per_project_businessautomationworkload.yml:169]: Please try to clean this up per previous comments. You can format it or you can increase the line length and keep it on a single line.

**Review by stencell — CHANGES_REQUESTED** [review]
There are a number for formatting corrections needed in this PR.

File: ansible/roles/ocp4-workload-pam-fraudmanagement-workshop/tasks/workload_per_project_amqstreamsworkload.yml (hunk 1 above)
Comment (ritesh97-rh) [reply ansible/roles/ocp4-workload-pam-fraudmanagement-workshop/tasks/workload_per_project_amqstreamsworkload.yml:16]: Sounds good, will clean up everywhere.

File: ansible/roles/ocp4-workload-pam-fraudmanagement-workshop/tasks/workload_per_cluster_gitea.yml [hunk 3]
```diff
@@ -33,7 +33,7 @@
   register: gitea
   retries: "{{ _retry }}"
   delay: "{{ _delay }}"
-  until: gitea.stdout == "True"
+  until: gitea.stdout == "true"
```
Comment (jkupferer) [inline ansible/roles/ocp4-workload-pam-fraudmanagement-workshop/tasks/workload_per_cluster_gitea.yml:36]: Can't change `True` to `true` in this context.

**Review by jkupferer — CHANGES_REQUESTED** [review]

**Pull request closed without merging**